### PR TITLE
Cypress commands for baseline resource creation/cleanup + Jobs E2E tests

### PR DESCRIPTION
**Pull request by vidyanambiar**
New Cypress commands:
1. `createBaselineResourcesForAWX`: Create either just an org, or an org + inventory + project + job template
    - polls projects API to wait till project has synced
2. `cleanupBaselineResourcesForAWX`: Cleans up baselines resources created

Initial tests for jobs UI

### CODE REVIEW

**Review by AlexSCorey — COMMENTED** [review]
This all looks really good, but I do worry about the recursive function polling the api for the project sync.

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -288,3 +286,126 @@ Cypress.Commands.add('selectRowInDialog', (name: string | RegExp, filter?: boole
...
+      });
+    });
+  });
+});
+
+// Polling to wait till a project is synced
+function waitForProjectToFinishSyncing(projectId: number) {
+  cy.requestGet<Project>(`/api/v2/projects/${projectId}`).then((project) => {
+    if (project.status === 'successful') return;
+
```
[the remark this answers is not in the document]
Comment (vidyanambiar) [reply cypress/support/commands.ts:357]: @AlexSCorey That's a good point and I agree. Perhaps we can use both options to limit the number of requests we make.

[the remark this answers is not in the document]
Comment (vidyanambiar) [reply cypress/support/commands.ts:358]: @AlexSCorey I've addressed this in the latest commits. I'm waiting 1 second between requests and limiting the max number of requests to 300 (assuming project syncs could take up to 5 min if the instance is under load from other jobs). I'm seeing the sync complete in about 6-7 seconds on average.

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -288,3 +287,160 @@ Cypress.Commands.add('selectRowInDialog', (name: string | RegExp, filter?: boole
...
+      // This will take care of deleting the project and the associated org, inventory
+      cy.deleteAwxProject(project);
+    });
+  }
+});
+
+let requestCount = 1;
+
+// Polling to wait till a project is synced
+function waitForProjectToFinishSyncing(projectId: number) {
```
Comment (AlexSCorey) [inline cypress/support/commands.ts:407]: I like this approach.  We can build on it/modify it if we need to in the future but I think this is the right approach for right now.

**Review by AlexSCorey — APPROVED** [review]

**Pull request merged**